packages/framework/assets/js/admin/components/order.js

```diff
@@ -66,7 +72,18 @@ export default class OrderPreview {
         this.isLoading = false;
         this.isLoaded = true;
         this.$previewBoxWindow.html(responseHtml);
-        this.$previewBoxWindow.show();
+        this.$previewBoxWindow.show(function () {
+            let tableHeight = $(this).parents('table').height();
```

Comment:
:+1: for `let`, but I think that it is `const` (like as `tablePosition` and `popupWindowPosition`) :smile:. When you accept my previous point, you should use something like above: `$(this).find('.js-*')`

Reply:
fixed